src/coreclr/src/jit/instrsarm64.h

```diff
@@ -378,6 +436,56 @@ INST3(mvn,     "mvn",    0, 0, IF_EN3I,   0x2A2003E0,  0x2A2003E0,  0x2E205800)
                                    //  mvn     Rd,(Rm,shk,imm)      DR_2F  X0101010sh1mmmmm iiiiii11111ddddd   2A20 03E0   Rm {LSL,LSR,ASR} imm(0-63)
                                    //  mvn     Vd,Vn                DV_2M  0Q10111000100000 010110nnnnnddddd   2E20 5800   Vd,Vn    (vector)
 
+//    enum     name     FP LD/ST            LS_2D        LS_3F        LS_2E
+INST3(ld1_2regs,"ld1",   0,LD, IF_EN3J,   0x0C40A000,  0x0CC0A000,  0x0CDFA000)
```

Comment:
I guess you couldn't figure out a way to merge these (ld1_2regs, ld1_3regs, ld1_4regs, st1_2regs, etc.) with their respective ld1/st1 definitions, above? Or create a single ld1_multiregs/st1_multiregs that is distinguished in code with insOpts or similar?

Reply:
Yes, I have considered different approaches while adding these instructions.

Since multiple registers variants exist only for a small number of instructions, namely `ld[1234]`, `ld[1234]r`,`st[1234]` ,`tbx` and `tbl` and only a subset of those require a way to specify a number of registers (since it's implied for  `ld[1234]r`, `ld[234]`, `st[234]`) I decided not to disrupt other emitter parts with adding number of registers to `insOpts`.